### DISCUSSION TITLE
Naming the default branch for repositories

### DISCUSSION THREAD

**Discussion by jcchavezs** [corazawaf/coraza#773] [General]
Currently repositories in the organization use two default branch names: `main` and `master` (notably coraza repo uses `v3/master`). We should aim to be consistent across repositories and hence settle either on one or the other as a consensus. Opening this discussions that should be finished in one week (as per agreed in the last coraza monthly meeting) for @corazawaf/core-developers (somehow @fzipi isn't in the team alias but I believe he should) to support one of them (you can also explain your reasons) or propose a new one.

**Comment by jcchavezs** [comment]
+1 to `main`

**Reply by anuraaga** [reply]
+1 to `main`. For Coraza, `main` and not `v3/main` will make it smoother for contribution I think.

**Reply by M4tteoP** [reply]
+1 to `main`

**Reply by jptosso** [reply]
+1 to `master`

**Comment by jptosso** [comment]
Btw, I used v3/dev and not dev because we plan to provide maintenance for older versions of Coraza. This means every time we want to patch something in old version we will have to clone a tag and create a new branch, creating an opportunity to make a mistake. That's why IMHO I think we should always use the major version as a branch prefix

**Reply by piyushroshan** [reply]
+1 to `main`

**Comment by jcchavezs** [comment]
It seems like `main` is the winner name. I will change repositories accordingly. Thanks everybody for participating.
